DTS-STN/passport-status
# Rename home page to status page

## [ADO-XXX](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/xxx)

### Description

Quick PR to rename `/home` to `/status` so it can properly be tracked by git as pointed out by @s-laugh in #35 

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/status` and ensure the form shows up properly

### Additional Notes
